app/views/wpcc/shared/_your_details.html.erb

```diff
@@ -11,4 +11,11 @@
       <%= link_to(t('wpcc.edit'), new_your_detail_path, class: 'section__heading-edit') %>
     </span>
   </h2>
+  <% if session[:manually_opt_in] %>
```

Comment:
This is the kind of thing I would use a helper for. Is it worth presenting piece of predicate functionality in another way, perhaps?

```
module Wpcc
  module YourDetailsFormHelper
    def require_ manual_opt_in?
      ...
    end
  end
end
```